modules/wkt/src/lib/encode-wkt.ts

```diff
@@ -38,24 +40,24 @@ export default function encodeWKT(gj: {
   }
 
   function wrapParens(s: string) {
-    return '(' + s + ')';
+    return `(${s})`;
   }
 
   switch (gj.type) {
     case 'Point':
-      return 'POINT (' + pairWKT(gj.coordinates as any[]) + ')';
+      return `POINT (${pairWKT(gj.coordinates as number[])})`;
```

Comment:
Nit: Since we've already defined a `wrapParens` helper, it might be clearer to use that? Right now since the expression inside the `${}` is a function, it's a little hard to tell that the `)` right before the closing `` ` `` is part of the string, and not the expression. Might be clearest IMO to have
```js
return 'POINT ' + wrapParens(pairWKT(gj.coordinates));
```